src/installation.md

```diff
@@ -1,98 +1,135 @@
-## Installation of the RMF Essentials
+## Install ROS 2 Galactic
```

Comment:
Can we keep the installation as generic as possible so we don't have to update two places every time there's a new ROS version? (as we already have the instructions on the readme. 

Maybe a good idea is to say here that the instructions are generic and we use galatic as example, and point somewhere to the README for the latest instructions?

Reply:
Got it will do

Reply:
> and point somewhere to the README for the latest instructions?

I didn't find this, can we still add a link to the README of the rmf repo saying there we have the latest install instructions?